src/test/javascript/spec/component/course/header-course.component.spec.ts

```diff
@@ -74,4 +76,46 @@ describe('Header Course Component', () => {
         expect(component.enableShowMore).toBeFalse();
         expect(component.courseDescription).toBe(courseWithShortDescription.description);
     });
+
+    it('should display manage button', () => {
+        component.course = courseWithShortDescription;
+        component.course.isAtLeastTutor = true;
+        const router = TestBed.inject(Router);
+        const urlSpy = jest.spyOn(router, 'url', 'get');
+        urlSpy.mockReturnValue('/some-url');
+
+        const showManageLectureButton = component.shouldShowGoToCourseManagementButton();
+        expect(showManageLectureButton).toBeTrue();
+    });
+
+    it('should not display manage button in course management', () => {
+        component.course = courseWithShortDescription;
+        component.course!.isAtLeastTutor = true;
+        const router = TestBed.inject(Router);
+        const urlSpy = jest.spyOn(router, 'url', 'get');
+        urlSpy.mockReturnValue('/course-management/some-path');
+
+        const showManageLectureButton = component.shouldShowGoToCourseManagementButton();
+        expect(showManageLectureButton).toBeFalse();
+    });
+
+    it('should not display manage button to student', () => {
+        component.course = courseWithShortDescription;
+        component.course!.isAtLeastTutor = false;
+        const router = TestBed.inject(Router);
+        const urlSpy = jest.spyOn(router, 'url', 'get');
+        urlSpy.mockReturnValue('/some-url');
+
+        const showManageLectureButton = component.shouldShowGoToCourseManagementButton();
+        expect(showManageLectureButton).toBeFalse();
+    });
```

Comment:
(optional) I think you could combine these tests in a parameterized test as there's quite a lot of repitition. I'd use three parameters: ``[ isAtLeastTutor, url, shouldBeVisible ]`` and then test all combinations. Search for previous occurrences of ``it.each`` to see how it works. :)

Reply:
Let's focus on the feature for now